permissions/Qr.permission

```diff
@@ -0,0 +1,11 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-qr
+# x-sailjail-description = QR
```

Comment:
Pondering about this short description... Somehow just "QR" sounds odd.

Reply:
I would maybe use these:
descripiton: QR codes
long-description: Read QR codes